src/toy_strategies/offer_maker/tutorial/OfferMakerTutorial.sol

```diff
@@ -3,13 +3,13 @@ pragma solidity ^0.8.10;
 
 // Import the types we will be using below
 import {Direct} from "mgv_src/strategies/offer_maker/abstract/Direct.sol";
-import {IMakerLogic} from "mgv_src/strategies/interfaces/IMakerLogic.sol";
+import {ILiquidityProvider_SDK as ISDK} from "mgv_src/strategies/interfaces/ILiquidityProvider_SDK.sol";
 import {IMangrove} from "mgv_src/IMangrove.sol";
 import {IERC20, MgvLib} from "mgv_src/MgvLib.sol";
 
 //----------------
 
-contract OfferMakerTutorial is Direct {
+contract OfferMakerTutorial is Direct, ISDK {
```

Comment:
Hm, for the tutorial the intend was to keep it simple so I left out a number of parameters that I did not want to explain.
Wouldn't that be O.K.?

Otherwise we can have this change and then we just have to state in the tutorial that they should not care about these just yet.

Also, new name... ISDK... hm - again consider spelling it out :p

Reply:
OK but it would be the occasion to mention we add these interface for a reason

Reply:
You decide - both can work IMHO.